server/main.lua

```diff
@@ -159,6 +170,13 @@ exports('CreatePlayerVehicle', createPlayerVehicle)
 ---@return boolean success, ErrorResult? errorResult
 local function setPlayerVehicleOwner(vehicleId, citizenid)
     assert(vehicleId ~= nil, "required field vehicleId was nil")
+    if not triggerEventHooks('changeVehicleOwner', { vehicleId = vehicleId, newCitizenId = citizenid }) then
+        return false, {
+            code = 'hook_cancelled',
+            message = 'a changeVehicleOwner event hook cancelled this operation'
+        }
+    end
+    triggerEventHooks('')
```

Comment:
why is this trigger empty?

Reply:
ditto, not sure this is needed @Manason ?

Reply:
removed.